### ISSUE TITLE
Storing phones.txt

### ISSUE THREAD

**Issue by danpovey** [kaldi-asr/kaldi#968]
Something along these lines off issue #965 that I'd like to be done, is to copy phones.txt to any model-training directory [and from there to any place we copy the models to].
There would also be a script with the following usage info:
```
utils/lang/check_phones_compatible.sh <phones-symbol-table1> <phones-symbol-table2>
e.g.: utils/lang/check_phones_compatible.sh data/lang/phones.txt exp/tri3/phones.txt
This script exits with status zero if the phone symbol tables are the same except
for possible differences in disambiguation symbols (meaning that all symbols except 
those beginning with a # are mapped to the same values).
Otherwise it prints a warning and exits with status 1.
For the sake of compatibility with older scripts that did not write the phones.txt to
model directories, this script exits silently with status 0 if one of the phone symbol
tables does not exist.
```
... and any scripts that take a lang directory and a model or alignment directory, and require the two to be compatible, would perform this check, in a way that's back compatible with scripts that didn't used to write phones.txt, something like:
`utils/lang/check_phones_compatible.sh $lang/phones.txt $dir/phones.txt`
if `set -e` was done, and
`utils/lang/check_phones_compatible.sh $lang/phones.txt $dir/phones.txt || exit 1`
otherwise.
Obviously this would require some kind of testing.

BTW, In general I don't like depending on the printed form of any symbols, but in this case it seems pretty safe, and easier than other alternatives.

**Comment by danpovey** [comment]
@LvHang, you could look at this.

**Comment by danpovey** [comment]
Was addressed in #1022
